lib/queue.js

```diff
@@ -141,6 +141,13 @@ var Queue = function Queue(name, redisPort, redisHost, redisOptions){
   // Create blocking client (used to wait for jobs)
   //
   this.bclient = createClient();
+  Object.defineProperty(this, 'jobWaitCount', { get: function() {
+    return this.bclient.commandQueue.length;
+  } });
+  Object.defineProperty(this, 'whenCurrentWaitFinished', { get: function() {
+    var currentMove = this.bclient.commandQueue.peekFront()
+    return currentMove && currentMove.command.promise || Promise.resolve();
+  } });
 
```

Comment:
Mostly so that they can be dynamic properties, rather than functions:
```javascript
console.log('we are waiting for', queue.jobWaitCount, 'jobs')
// we are waiting for 4 jobs
```
rather than
```javascript
console.log('we are waiting for', queue.getJobWaitCount(), 'jobs');
// we are waiting for 4 jobs
```

I also like that they're defined close to `queue.bclient` since they're basically just attributes of `queue.bclient` that I'm giving easier names to.

Would you prefer that I change them to methods on `Queue.prototype`?